packages/echo-base/src/__tests__/utils/storage.test.ts

```diff
@@ -53,7 +53,7 @@ describe('Storage', () => {
             });
             storage.setItem<string>('foo', 'bar');
 
-            expect(localStorage.setItem).toBeCalledWith('foo', 'bar');
+            expect(localStorage.setItem).toBeCalledWith('foo', JSON.stringify('bar'));
```

Comment:
The unit test should not know about JSON.stringify, that is internal logic for the storage. In the future we might change the localstorage to be something else, like indexDb, or encrypted storage.

unit test should be:
```
storage.setItem<string>('foo', 'bar');
const actual = storage.getItem("foo")
expect(actual).toBe("foo");
```
should be possible by mocking get and set of localStorage.setItem/getItem